notebooks/byod.py

```diff
@@ -98,27 +98,33 @@ def upload_columns(table: str, columns: Dict[str, List[Any]]):
             tsv_data += os.linesep + "\t".join([f"{i}", *[columns[h][i] for h in column_headers]])
         upload_data_table(tsv_data)
 
+    def parse_cram_crai(filename: str):
+        sample, _ = filename.split(".", 1)                                                                    
```

Comment:
This just grabs the first element:

```
>>> 'a.b.c'.split(".", 1)
['a', 'b.c']
```

So a file called `1.reads.NG12345.cram.crai` would just become `1`.  This should probably be changed to only remove any of `.cram`/`.crai`/`.cram.crai`.